cdap-docs/developer-manual/source/pipelines/creating-pipelines.rst

```diff
@@ -212,6 +212,74 @@ Two interfaces are available:
   underlying ``cron`` program. The details of that program will depend on the operating
   system used by the host of the CDAP Master process.
 
+With the CDAP Studio, you can also create a schedule that launches a pipeline when another pipeline
+reaches some certain statuses, and let the scheduled pipeline use properties from the triggering
+pipeline as runtime arguments. You can also create such schedule by using :ref:`HTTP request <http-restful-api-lifecycle-schedule-add>`.
+To specify how the properties from triggering Pipeline are used as runtime arguments in the scheduled Pipeline,
+you need to provide the stringified value of the following JSON object for the key
+"triggering.properties.mapping" in "properties" field in the schedule request body::
+
+    {
+      "arguments" : [
+        {
+          "source" : <runtime argument key in the triggering pipeline>,
+          "target" : <runtime argument key in the current pipeline>
+        },
+        ...
+      ],
+      "pluginProperties" : [
+        {
+          "stageName" : <stage name in the triggering pipeline>,
+          "source" : <plugin property key in the given stage in the triggering pipeline>,
+          "target" : <runtime argument key in the current pipeline>
+        },
+        ...
+      ]
+    }
+
+where "arguments" is a non-empty list of runtime arguments from the triggering pipeline
```

Comment:
this is really hard to understand. Use a concrete example with context.

For example, suppose you have pipeline 'X' in namespace 'nsX' and pipeline 'Y' in namespace 'nsY'. Pipeline 'X' is configured to write to a location set by the 'output' argument, and pipeline 'Y' is configured to read from a location set by the 'input' argument. You want to create a schedule so that pipeline 'Y' is run whenever pipeline 'X' completes, with pipeline 'Y' configured to read from the output of pipeline 'X'. In other words, if pipeline 'X' completes with argument 'output'='/path/to/data', you want pipeline 'Y' to run with argument 'input'='/path/to/data'. To do this, the 'triggering.properties.mapping' schedule property will be::

and the full HTTP PUT request body will be::